endpoints/ssh_keys.yml

```diff
@@ -20,4 +23,42 @@ get:
       content:
         application/json:
           schema:
-            $ref: '../components/schemas.yml#/DecoratedSSHKeys'
\ No newline at end of file
+            $ref: '../components/schemas.yml#/DecoratedSSHKeys'
+          examples:
```

Comment:
Are these examples meant to be here?

Putting the bundled output into SwaggerHub does not show what I'd expect:

![image](https://user-images.githubusercontent.com/46943/85608360-fb54bf80-b622-11ea-95e8-b2b79c1e4f52.png)

Reply:
Yes, those examples are meant to be there. You can't put multiple examples on a schema.

I bundled and looked locally. I'll upload to swaggerhub and look.

Reply:
Ah, ok, I wasn't sure how to make overriding work with `allOf`, etc. I was trying it out but it looks like it doesn't resolve correctly. Gonna have to use a different approach for these examples.

Reply:
That looks like the bundling didn't properly update the ref to a local component (notice the `../` in the path.